### PR TITLE
#42 Create app footer

### PR DESCRIPTION
**Pull request by YarkoSumyk**

### CODE REVIEW

File: src/configs/index.js [hunk 1]
```diff
@@ -64,3 +64,70 @@ export const HOMEPAGE_TITLES = {
...
+        item: 'Показати на мапі'
+      }
+    ]
+  },
+  1: {
+    title: 'Contacts',
+    items: [
+      { id: 1, url: '#', item: '+38 012 345 678' },
+      { id: 2, url: '#', item: 'horondi@gmail.com' },
+      { id: 3, url: '#', item: 'Lviv, Ugorska str.2' },
```
Comment (VolodymyrTrach) [inline src/configs/index.js:113]: It is just info. Why do you use the URL here?

File: src/containers/footer-links/footer-links.styles.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+export const useStyles = makeStyles(() => ({
+  cardBody: {
+    flex: '1 1 auto',
+    minHeight: '1px',
```
Comment (VolodymyrTrach) [inline src/containers/footer-links/footer-links.styles.js:6]: What will you see in 1px? make it more or remove it

Comment (YarkoSumyk) [reply src/containers/footer-links/footer-links.styles.js:6]: Removed

File: src/configs/index.js (hunk 1 above)
Comment (YarkoSumyk) [reply src/configs/index.js]: Removed

**Pull request merged**